### DISCUSSION TITLE
Where is the documentation?

### DISCUSSION THREAD

**Discussion by hwellmann** [OpenSlides/OpenSlides#6095] [Q&A] [answered]
I can't find any documentation for Openslides 3.3.

https://docs.openslides.org/ only covers 2.1 and 2.2.

Apart from user-level documentation, I'm interested in technical specifications and proofs that votes are secret and secure.

**Comment by tsiegleauq** [comment, marked as answer]
There is no _single_ documentation about OpenSlides >=3

There are some questions and answers:
https://support.openslides.com/help/en-gb

You can use compodoc to generate the client documentation
`npm run compodoc` (in client folder)

We have an assessment about our evoting
https://files.openslides.org/docs/Gutachten_Geheime-Wahlen-mit-OpenSlides3_20210325.pdf

**Reply by hwellmann** [reply]
Thanks for confirming that no comprehensive documentation is available. I had read that legal assessment, but what I require is a technical assessment.

Other systems like chvote or Belenios provide detailed specifications and even correctness proofs.

I will trust peer reviewed academic papers, but not an assessment by a lawyer contracted by the company behind OpenSlides.

**Reply by tsiegleauq** [reply]
>  Belenios provide detailed specifications and even correctness proofs

Never heard of thous. I'll look into it.

> I will trust peer reviewed academic papers, but not an assessment by a lawyer contracted by the company behind OpenSlides.

following that logic, you would also not trust a peer review if the company behind OS would issue one ;-)
If you can offer us a neutral peer review, feel free to do so. OS is MIT licensed after all.
